.github/RPC_PARITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "[automated] RPC parity test failure"
+labels: ["Bug"]
+---
+
+## Description
+
+Latest RPC parity test failed. Please [check the logs](https://github.com/ChainSafe/forest/actions/workflows/rpc-parity.yml) for more information.
```

Comment:
Could we be more specific and link to the failed run?

Reply:
Link added. See https://github.com/ChainSafe/forest/issues/5415